core/src/epicli/data/common/defaults/configuration/postgresql.yml

```
@@ -62,7 +62,9 @@ specification:
                 when: replication
   extensions:
     pgaudit:
-      enabled: no
+      enabled: yes
```

```suggestion
      enabled: no
```
Any reason to enable additional logging by default? This impacts performance.